src/handlers/assign.rs

```diff
@@ -280,8 +303,14 @@ async fn set_assignee(issue: &Issue, github: &GithubClient, username: &str) {
             .await
         {
             log::warn!("failed to post error comment: {e}");
+            return Err(e);
         }
     }
+
+    // Record the reviewer in the database
+    state.data.names.insert(username.to_string());
```

Comment:
We should handle normalization. Ideally with a newtype, but that's probably overkill here, the code already uses lowercase conversions all over the place.

Let's:
- Normalize the name to lowercase before putting it into the set
- Add a comment to the `names` field in `Reviewers` that explains that the names are normalized to lowercase
- Check `name_lower` instead of `reviewer_candidate.name` when computing `is_previously_assigned`